eclass/kernel-install.eclass

```diff
@@ -542,16 +542,15 @@ kernel-install_pkg_pretend() {
 
 	if ! use initramfs && ! has_version "${CATEGORY}/${PN}[-initramfs]"; then
 		ewarn
-		ewarn "WARNING: The standard Gentoo distribution kernel configuration"
-		ewarn "requires an initramfs to reliably boot! You have disabled the"
-		ewarn "initramfs USE flag, as a result, dracut was not pulled in as a"
-		ewarn "dependency. Please ensure that you are either overriding the standard"
-		ewarn "configuration or that an alternative initramfs generation"
-		ewarn "is installed and installkernel is configured to use it."
-		ewarn
-		ewarn "This is an advanced use case, you are on your own to ensure"
-		ewarn "that your system is bootable!"
+		ewarn "WARNING: The standard Gentoo distribution kernel configuration requires an"
+		ewarn "initramfs to reliably boot! You have disabled the initramfs USE flag, so"
+		ewarn "Dracut was not pulled in as a dependency."
```

Comment:
`Dracut was not pulled in` -> `no initramfs generator was pulled in`

Reply:
I'm trying to make the usage of "initramfs" as a kernel use flag more clear, as in setting this will make it pull dracut, but this error is also a result of not having an alternative set.

Reply:
But after this change that is no longer true, then `USE=initramfs` does not pull in dracut, it pulls in dracut *or* ugrd.